src/applications/hca/actions.js

```diff
@@ -1,23 +1,29 @@
+import appendQuery from 'append-query';
+import { apiRequest } from '../../platform/utilities/api';
+
 export const SUBMIT_ID_FORM_STARTED = 'SUBMIT_ID_FORM_STARTED';
 export const SUBMIT_ID_FORM_SUCCEEDED = 'SUBMIT_ID_FORM_SUCCEEDED';
 export const SUBMIT_ID_FORM_FAILED = 'SUBMIT_ID_FORM_FAILED';
 
-// eslint-disable-next-line no-unused-vars
 export function submitIDForm(formData) {
   return dispatch => {
-    dispatch({
-      type: SUBMIT_ID_FORM_STARTED,
+    dispatch({ type: SUBMIT_ID_FORM_STARTED });
+
+    const baseUrl = '/health_care_applications/enrollment_status';
```

Comment:
If you'd like to be able to test, you can change the URL to: `https://dev-api.va.gov/v0/health_care_applications/enrollment_status`. When entering data into the ID form, you can use the following test user to get a response with a valid status:

```
Wesley Ford
06/05/1986
796-04-3735
Male
```